### DISCUSSION TITLE
How to make Help Menu not take up whole screen view?

### DISCUSSION THREAD

**Discussion by frixaco** [sayanarijit/xplr#669] [Q&A] [answered]
Just found about `xplr` a couple of days ago and I'm in the processing switching to it full-time. Thank you very much for this amazing tool!

One very small "annoyance" I'm having is that whenever I e.g. try to create a file/directory Help Menu expands  and takes the whole screen. Same when I use `dual-pane` plugin too:

https://github.com/sayanarijit/xplr/assets/37442533/c54889f6-d58b-4962-999d-83bc60a80edf

Is there a way to make `xplr` not expand Help Menu and stay in bottom right?
For example, in the demo of  https://github.com/sayanarijit/dual-pane.xplr the Help Menu doesn't expand to whole screen view and stays in bottom right.

Here's my `.config/xplr/init.lua` just in case:
```
version = "0.21.3"

local home = os.getenv("HOME")
package.path = home .. "/.config/xplr/plugins/?/init.lua;" .. home .. "/.config/xplr/plugins/?.lua;" .. package.path

require("fzf").setup({
	mode = "default",
	key = "ctrl-f",
	bin = "fzf",
	-- args = "--preview 'pistol {}'",
	recursive = true, -- If true, search all files under $PWD
	enter_dir = true, -- Enter if the result is directory
})

require("dual-pane").setup({
	active_pane_width = { Percentage = 70 },
	inactive_pane_width = { Percentage = 30 }
})

```

**Comment by sayanarijit** [comment, marked as answer]
```lua
for name, mode in pairs(xplr.config.modes.builtin) do
  if mode.layout == "HelpMenu" then
    mode.layout = nil
  end
end

-- same for xplr.config.modes.custom
```

Something like this should work... typed on mobile, so there could be typos.

**Reply by frixaco** [reply]
@sayanarijit THAT WAS FAST. Thank you that perfectly works!
